src/diffusers/pipelines/pipeline_utils.py

```diff
@@ -340,7 +343,22 @@ def _get_pipeline_class(class_obj, config, custom_pipeline=None, cache_dir=None,
         return class_obj
 
     diffusers_module = importlib.import_module(class_obj.__module__.split(".")[0])
-    return getattr(diffusers_module, config["_class_name"])
+    pipeline_cls = getattr(diffusers_module, config["_class_name"])
+
+    if load_connected_pipeline:
```

Comment:
Yeah I need to write some better docs about it (will do in a follow-up PR). Essentially in case wants to load a connected pipeline via `DiffusionPipeline`